actions/utils.ts

```diff
@@ -85,15 +156,114 @@ function handlePromiseErrors<T>(
     .then(() => true)
     .catch((error) => {
       console.error(errorMessage, error);
-      return true;
+      return false;
     });
 }
 
 /**
- * Convenient utility to log in case theres an error writing in the registry
+ * Convenient utility to log in case theres an error writing in the registry and return a boolean with the result of the operation
+ *
  * @param registry Tenderly registry
  * @returns a promise that returns true if the registry write was successful
  */
-export function writeRegistry(registry: Registry): Promise<boolean> {
-  return handlePromiseErrors("Error writing registry", registry.write());
+export async function writeRegistry(): Promise<boolean> {
+  if (executionContext) {
+    return handlePromiseErrors(
+      "Error writing registry",
+      executionContext.registry.write()
+    );
+  }
+
+  return true;
+}
+
+var consoleOriginal = {
+  log: console.log,
+  error: console.error,
+  warn: console.warn,
+  debug: console.debug,
+};
```

Comment:
I have overridden the console, to make sure I can prevent from writing too big messages, which breaks Tenderly

Reply:
At the beginning i was only defining my own logger function, but since i had still issues with Tenderly removing logs, I became suspicious that some library was also logging something and this was breaking Tenderly,